src/meta.ts

```diff
@@ -0,0 +1,8 @@
+export const templates = [
```

Comment:
This is just preferred orderings of templates. Other templates added will go below these ones.